scenarios.md

```diff
@@ -262,6 +261,20 @@ A user may want to trust an artifact only if it is signed by a specific key. May
 
 1. Users must be allowed to configure specific trusted keys for specific artifacts or collections of artifacts.
 
+### Scenario #11: Revoking a Signature
+
+A signer determines that a signed artifact is no longer trusted. This could be a result of accidentally signing the wrong artifact, or a CVE discovered in the image and patched in a newer release. It should be possible for the signer to revoke the trust in that vulnerable artifact.
+
+1. A CVE is discovered in a signed artifact.
+1. The signature on the vulnerable artifact is revoked.
+1. Consumers that pulled the vulnerable artifact and verify the signature will be notified the signer no longer trusts the artifact.
+
+**Implications of this requirement:**
+
+1. Users that wish to still run artifacts that have their signatures revoked may need to take additional steps. Possible resolutions include configuring their client to ignore the revocation failure, disabling verification on the vulnerable artifact, or resigning the artifacts with a key they control and trust.
+1. Attackers attempting to replay old signatures to vulnerable artifacts should be blocked by the verification.
```

Comment:
> Attackers attempting to replay old signatures to vulnerable artifacts should be blocked by the verification.

It's not clear to me how this relates to revocation, if we say revocation is associated with the signature rather than the artifact.

Reply:
If you revoke a signature, and an attacker replays the revoked signature, perhaps on a poorly secured mirror, that should not be successful. Revocation shouldn't depend on a well behaved registry server.

Reply:
Ok got it, makes sense. Could we reword it so it's clearer - "replay revoked signatures" or "replay old signatures that have been revoked".

Reply:
Update pushed.